.cicd/test-package.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
```

Comment:
Consider another log folding group here to match the style of the other job steps.
```bash
echo '--- :evergreen_tree: Configuring Environment'
```